editor/app/ems/[incident_number]/page.tsx

```diff
@@ -90,8 +90,15 @@ export default function EMSDetailsPage({
    */
   const unmatchedTimeInterval: string[] = useMemo(() => {
     if (incident?.incident_received_datetime) {
-      // Return time interval only if we have any unmatched ems pcrs
-      if (ems_pcrs?.some((ems) => ems.crash_match_status === "unmatched")) {
+      // Return time interval only if the record doesnt have a crash_pk or any matched_crash_pks
+      if (
+        ems_pcrs?.some(
+          (ems) =>
+            ems.crash_pk === null &&
+            (ems.matched_crash_pks === null ||
+              ems.matched_crash_pks.length === 0)
```

Comment:
associated peoples record will now still display people w this time interval if they were unmatched and then set to unmatched by manual qa